js/5etools-spells.js

```diff
@@ -56,7 +56,7 @@ function d20plusSpells () {
 		if (toLoad.length) {
 			const handoutBuilder = !forcePlayer && window.is_gm ? d20plus.spells.handoutBuilder : d20plus.spells.playerImportBuilder;
 
-			const dataStack = (await Promise.all(toLoad.map(async url => DataUtil.loadJSON(url)))).flat();
+			const dataStack = (await Promise.all(toLoad.map(async url => DataUtil.spell.loadJSON(url)))).flat();
```

Comment:
as above, but worse, because this is loading the same set of spells _n_ times!

This entire thing can probably just be (untested):

```js
d20plus.importer.showImportList(
	"spell",
	await DataUtil.spell.loadJSON(),
	handoutBuilder,
	{
		groupOptions: d20plus.spells._groupOptions,
		forcePlayer,
		listItemBuilder: d20plus.spells._listItemBuilder,
		listIndex: d20plus.spells._listCols,
		listIndexConverter: d20plus.spells._listIndexConverter,
	},
);
```